app/build.gradle

```diff
@@ -180,6 +182,8 @@ dependencies {
     String espressoVersion = '3.4.0'
     String serialization_version = '1.4.0'
 
+    coreLibraryDesugaring 'com.android.tools:desugar_jdk_libs:1.2.2'
```

Comment:
I noticed a newer version of this library, would you please be able to update it?

Reply:
That requires AGP 7.4.0.